src/entities/Month.ts

```diff
@@ -12,3 +12,35 @@ export enum Month {
     NOVEMBER = "November",
     DECEMBER = "December",
 }
+
+export let months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+export let monthsToNumber = {
+    January: "01",
+    February: "02",
+    March: "03",
+    April: "04",
+    May: "05",
+    June: "06",
+    July: "07",
+    August: "08",
+    September: "09",
+    October: "10",
+    November: "11",
+    December: "12",
+};
```

Comment:
I always created copies of API/other non-visual stuff to work with, so the original functionality would not be lost during the refactoring. This might be one of the left-overs, I will look into this. 

Whilst the `monthToNumber` is certainly NOT needed, it's just cleaner imo to have it instead of manipulating the enum always in the Vue component. The way you suggested it, we would somehow have to make sure to always use 2 digits in order to get "09" and not only "9" which blurs the code even more.